plugin/src/main/java/me/champeau/gradle/igp/internal/git/cli/ExecOpsHelper.java

```diff
@@ -31,7 +31,7 @@ boolean isSuccess() {
 
     void assertNormalExitValue() {
       if (exitCode != 0) {
-        throw new IllegalStateException(String.format("Command failed with exit code %d: %s", exitCode, stdErr));
+        throw new IllegalStateException(String.format("Command failed with exit code %d: %s", exitCode, stdErr.get()));
```

Comment:
This was a straightforward error on my part (unfortunately quite easy to make with Gradle Providers).